spec/msk_cluster_spec.rb

```diff
@@ -15,4 +15,9 @@
     expect(instance.client).to receive(:get_bootstrap_brokers).with(cluster_arn: ACTUAL_MSK_ARN).and_call_original
     instance.bootstrap_broker_string
   end
+
+  it 'retrieves a sorted csv string listing bootstrap hosts' do
+    expect(instance.client).to receive(:get_bootstrap_brokers).with(cluster_arn: ACTUAL_MSK_ARN).and_call_original
```

Comment:
This will test the split and sort

```ruby
allow(instance).to receive(:bootstrap_broker_string).and_return("https://foo:9292,https://bar:8080")
expect(instance.sorted_bootstrap_broker_string).to eq "https://bar:8080,https://foo:9292"
```

Reply:
done and fixed.